mit_frontend/src/routers/UserInfo.js

```diff
@@ -143,120 +156,63 @@ class UserInfo extends Component {
               <MenuItem value={"56 - 60"} primaryText="56 - 60" />
               <MenuItem value={"Above 60"} primaryText="Above 60" />
             </DropDownMenu>
-            <p>Location</p>
-            <TextField
-              hintText="City"
-              floatingLabelText="City"
-              type="city"
-              floatingLabelFocusStyle={styles.textField.text}
-              underlineFocusStyle={styles.textField.underline}
-              value={this.state.city}
-              onChange={
-                (event) => {this.setState({city: event.target.value})}
-              }
-
-    
+            <p>City</p>
+            <div>
+              <Geosuggest
+                ref={el=>this._geoSuggest=el}
+                placeholder="Start typing!"
+                initialValue=""
+                fixtures={fixtures}
+                onSuggestSelect={this.onSuggestSelect}
+                location={new google.maps.LatLng(37.3382, 121.8863)}
+                radius="20" />
 
+              <button onClick={()=>this._geoSuggest.focus()}>Focus</button>
+              <button onClick={()=>this._geoSuggest.clear()}>Clear</button>
+              <button onClick={()=>this._geoSuggest.selectSuggest()}>Search</button>
```

Comment:
Not sure what the search button for. It seems not be able to clicked. Should we keep this?